webapp/src/components/form/FormCreator.js

```diff
@@ -0,0 +1,310 @@
+        {error}
+      </div>
+    )
+  }
+  return (
+    <>
+      <Prompt
+        when={!isSaved && !saved}
+        message="Some Changes have not been saved. Are you sure you want to leave without saving them?"
+        />
+      {homeRedirect && <Redirect to="/" />}
+      {errorResponse ? (
+        <div className='tooltiptext-error response-error'>
+          <Trans i18nKey='errorResponse' className='response-error'>{{errorResponse}}</Trans>
+        </div>
+      ) : (
+        <div className='application-form-wrap'>
+          <TopBar
+            title={setNominate ? "Application Form" : "Review Form"}
+            t={t} />
+          <div
+            style={{ textAlign: 'end', width: '61%' }}
+            className='add-section-btn-wrapper'
+            >
+            <button
+              className='add-section-btn'
+              data-title="Add Section"
+              onMouseUp={() => addSection()}
+              >
+              <i className="fas fa-plus fa-lg add-section-icon"></i>
+            </button>
+            <input
+              type="button"
+              value="Save"
+              style={!isReview ? { marginTop: '16em'} : {}}
+              className='save-form-btn'
+              data-title="Save"
+              onClick={handleSave}
+              disabled={isSaveDisabled || disableSaveBtn}
+              />
+          </div>
+          <div className="application-form-wrapper">
+            {setNominate && (
+              <div className="nominations-desc">
+                <input
+                  id="nomination-chck"
+                  className="nomination-chck"
+                  type="checkbox"
+                  checked={nominate}
+                  onChange={e => handleCheckChanged(e)}
+                />
+                <span htmlFor="nomination-chck" className="nomination-info">
+                  {t('Allow candidates to nominate others using this application form'
+                  + '(Users will be able to submit multiple nominations, including for themselves.'
+                  + ' If this option is unchecked, a candidate can only apply for themselves)')}
+                </span>
+              </div>
+            )}
+            {setNominate ? (
+              <div className="dates-container">
+                <span className="dates">
+                  {t('Application opens ') + ' :'}
+                  <span className="date">
+                    {`${dateFormat(evnt.application_open)}`}
+                  </span>
+                </span>
+                <span className="dates">
+                  {t('Application closes ') + ' :'}
+                  <span className="date">
+                    {`${dateFormat(evnt.application_close)}`}
+                  </span>
+                </span>
+              </div>
+            ) : (
+              <div className="dates-container">
+                <span className="dates">
+                  {t('Review opens ') + ' :'}
```

Comment:
same here, it would make sense to keep the review specific stuff in ReviewForm.js. Otherwise this will get especially tricky when we introduce a third form type!